GeneralsMD/Code/GameEngine/Source/Common/StateMachine.cpp

```diff
@@ -518,15 +518,19 @@ State *StateMachine::internalGetState( StateID id )
 
 	if (i == m_stateMap.end())
 	{
+		// TheSuperHackers @info helmutbuhler 26/04/2025
+		// These logs cause mismatch on Release when logging is enabled.
 		DEBUG_CRASH( ("StateMachine::internalGetState(): Invalid state for object %s using state %d", m_owner->getTemplate()->getName().str(), id) );
```

Comment:
Should this be included?

Reply:
DEBUG_CRASH is enabled when using the RTS_DEBUG_CRASHING cmake option, which is useful to get the assert message for Replay mismatch.